django/contrib/auth/tests/test_auth_backends.py

```diff
@@ -476,19 +477,19 @@ def setUp(self):
         self.user1 = User.objects.create_user('test', 'test@example.com', 'test')
         self.user1.save()
 
-    @override_settings(AUTHENTICATION_BACKENDS=(backend, ) +
-            tuple(settings.AUTHENTICATION_BACKENDS))
+    @override_settings(AUTHENTICATION_BACKENDS=[backend] +
+            settings.AUTHENTICATION_BACKENDS)
     def test_permission_denied(self):
         "user is not authenticated after a backend raises permission denied #2550"
         self.assertEqual(authenticate(username='test', password='test'), None)
 
-    @override_settings(AUTHENTICATION_BACKENDS=tuple(
-        settings.AUTHENTICATION_BACKENDS) + (backend, ))
+    @override_settings(AUTHENTICATION_BACKENDS=settings.AUTHENTICATION_BACKENDS +
```

Comment:
As Collin said, slightly longer lines are okay rather than awkward wrappings like this.